go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/convox/certbot
+
+go 1.18
```

Comment:
If you created an issue to [update go](https://github.com/convox/issues-private/issues/670) why is it set to 1.18?

Reply:
i added the go mod 
so it automatically picked the golang version i locally have
The code use the `vendor` folder it doesn't check the go mod

Reply:
I can not resolve the dependency using `govendor` that's why i initialize the go mod  for dependency
the pkg version are kept same as before

Reply:
okay, but if it's already on 1.18 we don't need the issue, do we need it?

Reply:
The docker still uses the old version, that's why i mention in the issue
may be i should rephrase it

Reply:
https://github.com/convox/certbot/blob/master/Dockerfile#L1